app/routes/_protected.send.confirm.tsx

```diff
@@ -61,15 +56,5 @@ export default function SendConfirmationPage() {
     );
   };
 
-  return (
-    <Page>
-      <PageHeader>
-        <PageBackButton to="/send" transition="slideDown" applyTo="oldView" />
-        <PageHeaderTitle>Confirm Payment</PageHeaderTitle>
-      </PageHeader>
-      <PageContent className="flex flex-col items-center gap-4">
-        {renderPageContent()}
-      </PageContent>
-    </Page>
-  );
+  return renderPageContent();
```

Comment:
I think that this function is not needed and we can just return directly from the component